scripts/binary/binary-cleanup.js

```diff
@@ -40,8 +40,9 @@ const getDependencyPathsToKeep = async (buildAppDir) => {
   const startingEntryPoints = [
     'packages/server/lib/plugins/child/require_async_child.js',
     'packages/server/lib/plugins/child/register_ts_node.js',
+    'packages/server/node_modules/@cypress/webpack-batteries-included-preprocessor/index.js',
```

Comment:
It annoys me that this dependency is duplicated in the binary, but it's been that way for a while and cleaning it up is out of scope for this PR.